### PR TITLE
Fix styling across Delivery Console

### LINKED ISSUE #210: Columns filter dropdown is visually broken
It looks like this:

image

We should fix it.

### PR DESCRIPTION
**Pull request by rehandalal**
This fixes a number of style issues across the console.

Fixes #210.

r?

Note: The edit/clone pages could still use some design love but don't look broken.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico?v=1">
```
Comment (peterbe) [inline public/index.html]: What is the point of this?

Comment (rehandalal) [reply public/index.html]: Oh I changed the favicon from the default create react app favicon and the old one is cached for me. Favicon cache is kind of weird so it really needs cachebusting. I opened up a CRA issue to hopefully fix this there, however I'll add a comment for now.

**Comment by rehandalal** [comment]
bors r=@peterbe

**Comment by peterbe** [comment]
If it's stuck in your browser just go straight to http://localhost:3000/favicon.ico and hit refresh. That's how I got a new balloon favicon.

**Comment by rehandalal** [comment]
> If it's stuck in your browser just go straight to http://localhost:3000/favicon.ico and hit refresh. That's how I got a new balloon favicon.

I tried that. It didn't work.

**Pull request merged**